docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+web:
+  build: .
+  command: mix phoenix.server
+  environment:
+    PORT: 80
+  volumes:
+  - .:/app
+  ports:
+  - "80:80"
```

Comment:
Port 80 means you're going to have problems if you run this at the same time as Gravity in Docker locally.